Documentation/ceph-filesystem-crd.md

```diff
@@ -94,6 +96,16 @@ spec:
     activeStandby: true
 ```
 
+**IMPORTANT**: For erasure coded pools, we have to create a replicated pool as the primary pool and erasureCoded as the secondary pool.
+Separately, a directory layout property needs to be added to use the secondary EC data pool.
+
+```console
+mkdir /mnt/cephfs/myssddir
+$ setfattr -n ceph.dir.layout.pool -v <ec-pool-name> /mnt/cephfs/myssddir #example ec-pool-name, myfs-ec
```

Comment:
Rook generates the pool name, I believe in this case it would be the following for this example
```suggestion
$ setfattr -n ceph.dir.layout.pool -v myfs-ec-data1 /mnt/cephfs/myssddir 
```